.github/workflows/release.yml

```
@@ -0,0 +1,41 @@
+jobs:
+  build-wheels:
+    name: upload pybamm-cookie wheel artifacts
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout pybamm-cookie
+      - uses: actions/checkout@v4
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.12
+
+      - name: Build package
+        run: pipx run build
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v4.3.6
+        with:
+          name: pybamm_cookie_wheel
+          path: ./dist/*.whl
+          if-no-files-found: error
```

```suggestion
          name: sdist_and_wheel
          path: ./dist/*
          if-no-files-found: error
```

`pipx run build` creates both the sdist and the wheel, so why not upload both?